test/test_autotuner.cc

```diff
@@ -89,6 +89,25 @@ struct ATenCompilationUnitTest : public ::testing::Test {
   }
 };
 
+// TODO: see #124
+TEST_F(ATenCompilationUnitTest, DISABLED_Graph2) {
```

Comment:
I'd prefer this as a non-autotuner test.  The problem is not specific to the autotuner, and introducing a ~1min test time does not look like a good idea.

Reply:
@ftynse where shall this go? in particular, do we have a test file where we put such test cases? if not, I can create one.